pages/linux/equery.md

```diff
@@ -0,0 +1,23 @@
+# equery
+
+> View information about portage packages.
+
+- List all installed packages:
+
+`equery list '*'`
+
+- Search for installed packages:
+
+`equery list -po {{package_name}}`
```

Comment:
What do the -p and -o stand for? It would be nice to integrate the words they correspond to into the command description, to ease memorization.